package.json

```diff
@@ -21,12 +21,12 @@
     "format": "prettier . --write",
     "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
     "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
-    "generate-schemas": "deno run -A src/lib/schemas/generate.ts",
```

Comment:
Do we use this anymore?

Reply:
Nope, that's left-over from the pre-leaf stuff.